src/combinatorics/quiver/basic.lean

```diff
@@ -90,12 +90,13 @@ def comp {U : Type*} [quiver U] {V : Type*} [quiver V] {W : Type*} [quiver W]
 lemma comp_assoc
   {U V W Z : Type*} [quiver U] [quiver V] [quiver W] [quiver Z]
   (F : prefunctor U V) (G : prefunctor V W) (H : prefunctor W Z) :
-  (F.comp G).comp H = F.comp (G.comp H) :=
-begin
-  apply prefunctor.ext, rotate,
-  { rintro X, refl, },
-  { rintro X Y Z, refl, }
-end
+  (F.comp G).comp H = F.comp (G.comp H) := rfl
+
+infix ` ⥤q `:50 := prefunctor
+
+infix ` ⋙q `:50 := prefunctor.comp
+
+notation `𝟭q` := id
```

Comment:
Is that really worth having? It's not shorter and quite longer to type.

Reply:
I'm on the fence. I think it's worth having just for the sake of having the complete set.